tests/conftest.py

```diff
@@ -340,4 +340,4 @@ def sessionFixture():
 @fixture
 def contractsFixture(sessionFixture):
     sessionFixture.resetSnapshot()
-    return sessionFixture
+    return sessionFixture
```

Comment:
Style: Files should all end in a newline.  Recommend adding EditorConfig support to whatever editor you are using so this happens automatically.